app/assets/javascripts/base.js.erb

```diff
@@ -125,11 +125,13 @@ var instantClick
     if (newUrl) {
       history.pushState(null, null, newUrl.replace("?samepage=true","").replace("&samepage=true",""))
 
-      var hashIndex = newUrl.indexOf('#')
-        , hashElem = hashIndex > -1
-                     && document.getElementById(newUrl.substr(hashIndex + 1))
-        , offset = 0,
-        samePage = newUrl.indexOf("samepage=true") > -1
+      var hashIndex = newUrl.indexOf('#'),
+          hashElem = hashIndex > -1 && (
+            document.getElementById(newUrl.substr(hashIndex + 1)) ||
+            document.querySelector('[name='+newUrl.substr(hashIndex + 1))+'].anchor')
```

Comment:
Thank you for this PR @Link2Twenty ! I wasn't able to load the app locally because there's a syntax error on this line. I believe you were trying to do the following?

```suggestion
            document.querySelector(`[name='${newUrl.substr(hashIndex + 1)}'].anchor`)
```

Reply:
Well that's embarrassing 😅